ports/psoc6/modules/machine/machine_adcblock.c

```diff
@@ -23,6 +23,39 @@ const adc_block_channel_pin_map_t adc_block_pin_map[] = {
 /******************************************************************************/
 // MicroPython bindings for machine.ADC
 
+// Private helper function to get channel number for provided pin
+int16_t _get_adc_channel_number_for_pin(uint32_t pin) {
+    for (int i = 0; i < (sizeof(adc_block_pin_map) / sizeof(adc_block_pin_map[0])); i++)
```

Comment:
adc_block_pin_map has "PIN_P10_0". This needs to be changed no as the definition is no more available or?

Reply:
Yes, we have to check that according to how we integrate that part :)